src/core/drive/progress_bar.js

```diff
@@ -64,14 +64,14 @@ export class ProgressBar {
   }
 
   installProgressElement() {
-    this.progressElement.style.width = "0"
+    this.progressElement.style.width = "0"|"0"
```

Comment:
Hello, this seems to change the type from string to number. Perhaps the previous version should be kept here.